wgengine/magicsock/legacy.go

```diff
@@ -34,7 +33,7 @@ var (
 	errDisabled       = errors.New("magicsock: legacy networking disabled")
 )
 
-func (c *Conn) createLegacyEndpointLocked(pk key.Public, addrs string) (conn.Endpoint, error) {
+func (c *Conn) createLegacyEndpointLocked(pk key.Public, addrs []netaddr.IPPort, rawdst string) (conn.Endpoint, error) {
```

Comment:
rawDst or rawDest, but add a comment about what it is and what form it takes (comma-separated something? IPs, IP:ports?)

Reply:
Done. (The answer is: don't look inside it. Just return it from addrSet.DstToString.)